test/scopeSync.test.ts

```diff
@@ -0,0 +1,152 @@
+jest.mock('../src/js/wrapper', () => jest.requireActual('./mockWrapper'));
+import * as SentryCore from '@sentry/core';
+import { Scope } from '@sentry/core';
+import type { Breadcrumb } from '@sentry/types';
+
+import { enableSyncToNative } from '../src/js/scopeSync';
+import { getDefaultTestClientOptions, TestClient } from './mocks/client';
+import { NATIVE } from './mockWrapper';
+
+jest.mock('../src/js/wrapper');
+
+describe('ScopeSync', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('scope apis', () => {
+    let scope: Scope;
+
+    beforeEach(() => {
+      scope = new Scope();
+      enableSyncToNative(scope);
+    });
+
+    describe('addBreadcrumb', () => {
+      it('it only syncs once per scope', () => {
+        enableSyncToNative(scope);
+        enableSyncToNative(scope);
+
+        scope.addBreadcrumb({ message: 'test' });
+
+        expect(NATIVE.addBreadcrumb).toBeCalledTimes(1);
```

Comment:
Can we test the original call from the Scope were also called just to make sure we didn't miss calling them?